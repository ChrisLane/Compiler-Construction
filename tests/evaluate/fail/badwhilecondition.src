// While conditions should result in a boolean. This should not evaluate.
main() {
    while (9001) {};
}
