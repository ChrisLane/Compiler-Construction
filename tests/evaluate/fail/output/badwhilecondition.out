Fatal error: exception Failure("Invalid value for while condition.")
